lms/djangoapps/courseware/views/views.py

```diff
@@ -1787,3 +1787,4 @@ def check_access_to_course(request, course):
     # Redirect if the user must answer a survey before entering the course.
     if must_answer_survey(course, request.user):
         raise CourseAccessRedirect(reverse('course_survey', args=[unicode(course.id)]))
+        
```

Comment:
please fix the no newline at the end of file

Reply:
I added the line at the end of the file